Rakefile

```diff
@@ -36,6 +36,9 @@ end
 # If 1, then we link against LuaJIT. If 0, we use classic Lua VM.
 $doBuildJIT=1
 
+# If 1, then LUA GC profiling code is enabled
+$doEnableLuaGcProfile=0
```

Comment:
Yes -  I mean this flag's value should be based on the value here: https://github.com/LoomSDK/LoomSDK/blob/JG-LuaJIT-Profiling/Rakefile#L11